rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1249,14 +1249,12 @@ void spu_thread::do_dma_transfer(const spu_mfc_cmd& args)
 
 			if (!thread)
 			{
-				fmt::throw_exception("RawSPU not found (cmd=0x%x, lsa=0x%x, ea=0x%llx, tag=0x%x, size=0x%x)" HERE, args.cmd, args.lsa, args.eal, args.tag, args.size);
+				// Access Violation
```

Comment:
I didn't get that part, if you want to keep it UB if thread==nullptr, why keep the check at all?

Reply:
I want to unify all SPU access violations in av handler because they are "special", and past examples were made for it as well. This is why SPU sccess violations are errors and fatals in log.